test/functional/test-dom.js

```diff
@@ -148,4 +148,44 @@ describe('DOM', () => {
     expect(dom.childElementByAttr(parent, 'attr12', '3')).to.be.null;
     expect(dom.childElementByAttr(parent, 'attr3')).to.be.null;
   });
+
+  describe('contains', () => {
+    let connectedElement;
+    let connectedChild;
+    let disconnectedElement;
+    let disconnectedChild;
+
+    beforeEach(() => {
+      connectedElement = document.createElement('div');
+      connectedChild = document.createElement('div');
+      disconnectedElement = document.createElement('div');
+      disconnectedChild = document.createElement('div');
+
+      connectedElement.appendChild(connectedChild);
+      disconnectedElement.appendChild(disconnectedChild);
+      document.body.appendChild(connectedElement);
+    });
+
+    afterEach(() => {
+      dom.removeElement(connectedElement);
+    });
+
+    it('should use document.contains when available', () => {
```

Comment:
Updated title. We basically care that where it's used - it works. Polyfil is tested separately.